src/twisted/protocols/_smb/core.py

```diff
@@ -0,0 +1,519 @@
+# -*- test-case-name: twisted.protocols._smb.tests -*-
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+"""Implement Microsoft's Server Message Block protocol"""
+
+import struct
+import binascii
+import uuid
+import time
+from collections import namedtuple
+
+from twisted.protocols._smb import base, security_blob
+from twisted.protocols._smb.interfaces import (ISMBServer, IFilesystem, IPipe,
```

Comment:
To avoid having a bunch of `interfaces.py` files and the potential import confusion that might provoke, we have generally gone with `<whatever>.i<whatever>` for interface module names; so in this case, `from twisted.protocols._smb import ismb`